Makefile

```diff
@@ -133,7 +134,7 @@ uninstall:
 
 clean:
 	@rm -v -f $(DISTTARGET) vim-src/.config.h-modified vim-src/src/*.orig
-	@cd vim-src && $(MAKE) -s distclean && hg update -C
+	@cd vim-src && $(MAKE) -s distclean && ( git reset --hard; git clean -x -f -d -q; )
```

Comment:
Please write the code in such a way that the exit status for git is validated. If you look at static-tmux, you'll see that I have `set -e` inside the conditional which will cause the command execution to stop if there's an unhandled error.